src/ui/Toggle/index.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../styles/variables';
+
+.sendbird-input-toggle-button {
```

Comment:
have recommendation to rewrite this file, will explain

Reply:
I applied that!